bin/gcs-deploy.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+
+set -ex
+ EXTENSION_WORKSHOP_BUCKET_GCS=$1
+
+ echo $EXTENSION_WORKSHOP_BUCKET_GCS
```

Comment:
Is there a reason for this script to be slightly indented?

Reply:
No, reason could just a linting problem in my vs code set up.